apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -93,6 +94,7 @@ function LoginPage(props: LoginPageProps) {
                   preparePostSSORedirect()
                 }}
                 twoFactorAuthenticationRequired={twoFactorAuthSSOErrorResponse}
+                twoFactorAuthResetUri={`${window.location.origin}${RESET_2FA_ROUTE}?${RESET_2FA_SIGNED_TOKEN_PARAM}=`}
```

Comment:
The email to disable 2FA will contain this address. For `SageAccountWeb`, that should be the `/reset2FA` route.